defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-aptly_gpg_key_dir: "{{ lookup('env', 'HOME') }}"
+aptly_gpg_key_dir: "aptly"
```

Comment:
it's been a while, talk me through this one?

Reply:
Hi @aioue, thanks for reviewing my PR. Sure, here I just set the directory to "aptly" rather than looking at the username in the environment variable. While I though using the lookup was slick, I ran into trouble because I had a username difference between my host machine and the ansible target.